frontend/src/pages/Landing/Payments/LandingPaymentsPage.tsx

```diff
@@ -106,22 +230,115 @@ export const LandingPaymentsPage = (): JSX.Element => {
           />
         </SimpleGrid>
       </LandingSection>
+      <FeatureSection
```

Comment:
Nit: for readability sake, do you think it will be worth abstracting the different parts of the page into its own component:

The main header
Payment features
Help center ...

Reply:
Refactored as much as possible in [d16a604](https://github.com/opengovsg/FormSG/pull/6462/commits/d16a60426584394a2e78a4ace02365967299d2f9)!